docs/changelog.md

```diff
@@ -1,4 +1,19 @@
 # TBD
+### Features
```

Comment:
Dude idk what happened but your English took a huge leap up with this entire changelog!!! Really really nice; it looks almost native!

Reply:
Also I LOVE how detailed your remediation steps were

Reply:
tyty! I have good teachers :smile: